src/subcircuit/convertSub.py

```diff
@@ -38,12 +38,12 @@ def createSub(self):
             else:
                 self.msg = QtGui.QErrorMessage(None)
                 self.msg.showMessage(
-                    'The subcircuit does not contain any Kicad netlist\
-                     file for conversion.')
+                    'The subcircuit does not contain any Kicad netlist file'
+                    + 'for conversion.')
                 self.msg.setWindowTitle("Error Message")
         else:
             self.msg = QtGui.QErrorMessage(None)
             self.msg.showMessage(
-                'Please select the subcircuit first. You can either create \
-                new subcircuit or open existing subcircuit')
+                'Please select the subcircuit first. You can either create'
+                + 'new subcircuit or open existing subcircuit')
```

Comment:
```suggestion
                + ' new subcircuit or open existing subcircuit')
```
require space between create and new